pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
@gfyoung your comments indicate you don't understand my point. We *don't* have tests that have a ``levels`` attribute IN A PICKLE. Thus the tests don't test the correct thing, and you cannot assure back-compat.